packages/flatbuffers/src/schema/root.fbs

```diff
@@ -12,6 +12,15 @@ union Type {
   Blob
 }
 
+enum ComparisonRuleMode:ubyte { Absolute, Relative }
```

Comment:
I see your point. Let's talk about this 1:1. I'd slightly prefer to make the distinction in TS to avoid the complexity overhead of the flatbuffers schema.